q

```diff
@@ -0,0 +1,111 @@
+[1mdiff --git a/index.js b/index.js[m
```

Comment:
Yeah, please delete file q.  Didn't intend to create, let alone commit.
Thanks.
On May 21, 2015 6:16 PM, "DRTforGSA18F" <notifications@github.com> wrote:

> In q
> <https://github.com/DRTforGSA18F/marketapi/pull/5#discussion_r30853799>:
>
> > @@ -0,0 +1,111 @@
> > + [1mdiff --git a/index.js b/index.js [m
>
> @chriszeiders <https://github.com/chriszeiders>, I will get latest and
> delete this file unless you see any purpose to it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/DRTforGSA18F/marketapi/pull/5/files#r30853799>.
>